setup.py

```diff
@@ -15,6 +15,7 @@
 
 
 install_requires = [
+    'Keras>=2.1.6,<2.4'
```

Comment:
This should not be added here, since Keras is not a direct dependency of mlblocks.
Also note that current MLPrimitives master already fixed this problem.

Reply:
Updated the test dependency to use MLPrimitives master to fix this.

Also I apologize for the multiple commits/pushes, tests were passing just fine locally as they were using different versions of dependencies. I capped the issues that arise from newer versions.